src/config/defaults/db.php

```diff
@@ -41,6 +41,11 @@
      */
     'schema' => 'public',
     /**
+     * The database credentials in a string format following the standard uri syntax.
+     * Format: <driver>://<user>:<password>@<server>:<port>/<database>
```

Comment:
Much like the 'dsn' setting where you can just explicitly define the whole DSN string, we should probably note that if you set this to anything other than `null`, Craft will ignore all of the other DSN related config setting.